package.json

```diff
@@ -67,6 +67,16 @@
 				"title": "File Access: Make Read Only",
 				"enablement": "isWorkspaceTrusted"
 			},
+			{
+				"command": "readOnly.makeWriteableRecursive",
+				"title": "File Access: Make Writeable Recursive",
+				"enablement": "isWorkspaceTrusted"
+			},
+			{
+				"command": "readOnly.makeReadOnlyRecursive",
+				"title": "File Access: Make Read Only Recursive",
```

Comment:
Both commands, which will be available only in the Context Menu, should not have the `File Access:` prefix. Please, remove it.